cfpb/consumerfinance.gov
# Swap to using the AND operator for ask-cfpb

The existing ES implementation uses an AND operator for multiple words within a query, this will mirror the functionality for ES7. 


---

## Changes

- Swap to specifying the operator type for our search query.


## How to test this PR

1. Search a phrase such as "credit monitoring service" and confirm only results with the full phrase and not parts of the phrase.


## Screenshots
![image](https://user-images.githubusercontent.com/16838497/100017657-3f521900-2da9-11eb-897a-a9993ac6d082.png)


## Notes and todos

- Needs to be implemented across all searches potentially.


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
